net/extras/genInventoryFile.py

```diff
@@ -54,6 +54,9 @@ def writeInventory(self, outFd):
             outFd.write(var_line)
             var_line = "netplugin_if={} ".format(connInfo[node]['data'])
             outFd.write(var_line)
+            role = connInfo[node].get('role', 'worker')
```

Comment:
@neelimamukiri: As I understand, the inventory host vars has a lesser precedence over the role vars. So, this would not take into effect. Could you please verify this?